src/App.js

```diff
@@ -1,16 +1,48 @@
 import React from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import { useState } from 'react';
+import ChatLog from './components/ChatLog.js';
+// import InfoBar from './components/InfoBar.js';
 
 const App = () => {
+  const [entries, setEntries] = useState(chatMessages);
+  const [hearts, setHearts] = useState(0);
```

Comment:
So organized, setting all the state variables at the top of the component! Love to see it 💖